.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: ruby
 os: osx
-env: OSX=10.11
-osx_image: xcode7.3
-rvm: system
+env: OSX=10.10
+osx_image: xcode6.4
+rvm: 1.8.6-p287
```

Comment:
Why can't this stay at `system`, are you “simulating” 10.5 with this? And is the `OSX` variable actually doing anything?

Reply:
> are you “simulating” 10.5 with this

Kinda, yes.



> And is the OSX variable actually doing anything?

It's describing the version in the image for quick reference in the Travis UI.

Reply:
It's describing the version in the image for quick reference in the Travis UI.

> Ah, I see.